homeassistant/components/plugwise/sensor.py

```diff
@@ -257,7 +258,8 @@
     SensorEntityDescription(
         key="illuminance",
         name="Illuminance",
-        native_unit_of_measurement=UNIT_LUMEN,
+        native_unit_of_measurement=LIGHT_LUX,
```

Comment:
It might have missed something, but where is this implemented upstream?

As in, it is changing from lumen to lux.  Where does this happen? And how is this converted?

Reply:
When `lm` is not the correct u_o_m, let's change it to `lx`.

Reply:
> This was merged via #84386

That statement is incorrect. #84386 removed the device class. It did not change the unit of measurement, nor did it change the value.

Do you realize that lumen & lux are not the same things?

This PR is basically setting the celsius unit on a Fahrenheit temperature sensor.
The question I have is:  Where did the lumen change to lux in the code? Where is the value converted/changed?

Reply:
Reason for this change: `lm` is incorrect, the sensor is for illuminance, not for luminous flux.

Reply:
Checked the core appliances, and the unit is indeed, `lx` in the XML as well.

This is fixing a bug, unrelated to #84386, it corrects a unit. Please note, this is breaking change.

Please mark the PR as a breaking change and add a breaking change section to the PR description.

Reply:
Ok, all feedback implemented.

Reply:
No conversion necessary, using the unit `lm` was a mistake that was made in the past.